### PR TITLE
add option track mouse when zooming

### PR DESCRIPTION
**Pull request by zyansheep**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -75,13 +87,15 @@ fn camera_movement(
 pub struct PanCam {
     pub grab_buttons: Vec<MouseButton>,
     pub enabled: bool,
+    pub track_mouse: bool,
```
Comment (johanhelsing) [inline src/lib.rs:90]: I wonder if we can find a better name for this? zoom_to_pointer? cursor/mouse?

Currently, it's kind of hard to guess what it is from only the name.

**Review by johanhelsing — APPROVED** [review]
Nice!

No comments except naming. Change it if you have a better suggestion, if not I'll merge as is.

File: src/lib.rs (hunk 1 above)
Comment (zyansheep) [reply src/lib.rs:90]: Sure

Comment (zyansheep) [reply src/lib.rs:90]: zoom_to_cursor sounds like a better name

**Comment by zyansheep** [comment]
It seem github closed this because I force-pushed a channel... I opened a new one

**Pull request closed without merging**